### ISSUE TITLE
rhash falls back to printing non-relative paths in output

### ISSUE THREAD

**Issue by kralo** [rhash/RHash#135]
why does the full path get printed intermittently ?
Is it because of special chars?

Why would it not output relative path?

```
>rhash --version
RHash v1.4.0
```

```
Command line is:
rhash --sha256 -r -o D:\logs\rhash_1.txt .\
```

```
c75d284885a232000001e9a567a1cd715188ec743687a67e6a303c3be72a7e9c  kleee\prio1\download\root\migrat1_2007_10_ende\$RECYCLE.BIN\S-1-5-21-123456789-4172760481-734191380-1001\$R0S6Y7T\472\4723\4723-Kundendaten\4700_Niet_Maxooo-10000001_V5P_00_1.sldprt
183ad36be29a890123422d20f58de0804ce49d5652f22c4f098fe5b978969b40  C:\backups\kleee\prio1\download\root\migrat1_2007_10_ende\$RECYCLE.BIN\S-1-5-21-123456789-4172760481-734191380-1001\$R0S6Y7T\472\4723\4723-Kundendaten\4700_Schraube M10 (aehnlich ISO 8677)-10000001_V5P_00_1.sldprt
367ab89a4902872e54664aff9d123456000006f6c68964c31846e290f11766ee  kleee\prio1\download\root\migrat1_2007_10_ende\$RECYCLE.BIN\S-1-5-21-123456789-4172760481-734191380-1001\$R0S6Y7T\472\4723\4723-Kundendaten\4700_Schweissmutter_M8-10000000_V0P_00_1.sldprt
```

**Comment by rhash** [comment]
Thank you for reporting this bug!

The problem was caused by incorrect handling of long paths on Windows, which require UNC prefix.

The bug is fixed in the master branch by commits 55301b8ddcd147b02bd1efd3f7bd2f5ce9843c3f and 824a520cef1efda01494e76e726aae9fa71fab6b.

**Comment by rhash** [comment]
The fix is included into RHash v1.4.1.
